src/services/game.online.service.mjs

```diff
@@ -29,7 +29,7 @@ export const createGameOnline = async ({
   const gamesListRef = ref(database, "games");
   const gameRef = push(gamesListRef);
   set(gameRef, game);
-  return gameRef;
+  return gameRef.key;
 };
 
 export const joinGameOnline = async (gameId) => {
```

Comment:
Wir müssen hier noch die scores initieren, weil dies nicht mehr in createGame passiert oder wobei das passiert über die Laufzeit und das Modal wird eh erst am Ende angezeigt. Mhhh... sauber wäre es aber glaube ich die scores hier schon mit 0 zu setzen sobald ein user joined

Reply:
Muss bei leave auch wieder gelöscht werden dann